roles/dropwizard-api/tasks/gen-certs.yml

```diff
@@ -0,0 +1,39 @@
+---
+  - name: copy gen-certs.sh to dest
+    template:
+      src: gen-certs.j2
+      dest: /apis/gen-certs.sh
+    become_user: osu_apis
+
+  - name: stat gen-certs.sh
+    stat:
+      path: /apis/gen-certs.sh
+    register: gen_certs_stat
```

Comment:
Ah, right. The purpose of the ansible lint warning is to avoid running a command that might not be necessary. But we really do want to run gen-certs.sh every time (right?). So i would just add a comment to the task to tell ansible-lint to back off. https://github.com/willthames/ansible-lint#false-positives